APIs/openEO/large_scale_processing.qmd

```diff
@@ -59,40 +57,40 @@ df = gpd.read_file("https://artifactory.vgt.vito.be/auxdata-public/grids/LAEA-20
 
 ## Tuning your processing job
 
-Before kicking off large processing, you want to be very sure that the correct output is generated and that you have sufficient credits and resources available to finish your job in time. This can be done by simply running various jobs and using the statistics reported in the metadata to determine average parameters. (The map production section below shows a way to collect these parameters in a CSV.)
+Before starting large-scale processing, users should verify that the correct output is generated and that they have sufficient credits and resources to complete their jobs. This can be achieved by running test jobs and analyzing the statistics reported in the metadata to determine average parameters. (Refer to the map production section below for guidance on collecting these parameters in a CSV format.)
 
-For instance, for the case of processing the EU27 croptype map, consisting of ~11000 20km tiles, we made the following calculations up front:
+For example, when processing the EU27 croptype map, consisting of ~11000 20km tiles, we conducted the following upfront calculations:
 
-- The average runtime was 30 minutes, which means that it would take ~15 days of continuous processing with 15 parallel jobs.
-- The average cost was below 100 credits, so we would be able to process with a budget of 1100000 credits.
+- The average runtime per job was 30 minutes, indicating it would require approximately 15 days of continuous processing with 15 parallel jobs.
+- The average cost per job was below 100 credits, ensuring we could feasibly process within a budget of 1,100,000 credits.
 
-To achieve these numbers, we did have to optimize batch job settings and the overall workflow to reduce resource usage.
+We had to optimize batch job settings and streamline the overall workflow to achieve these performance metrics.
 
-A common bottleneck to parallelization is memory consumption, and it can be useful to know the maximum memory allocation on a single machine in your backend of choice. For instance, in a cloud environment with 16GB per machine and 4 CPUs, using slightly less than 4GB per worker is efficient as you can fit 4 parallel workers on a single VM while requiring 6GB would fit only 2 workers and leave about 4GB unused.
+A common challenge in parallelization is managing memory consumption, especially understanding the maximum memory allocation per machine in the chosen backend environment. For example, in a cloud setup where each machine has 16GB RAM and 4 CPUs, efficient usage would involve allocating less than 4GB per worker. This configuration allows four parallel workers to be fitted on a single VM. Conversely, requiring 6GB per worker would accommodate only two workers per VM, leaving approximately 4GB unused.
```

Comment:
Here we would truly benefit of more clear information for users. Best to align with Jeroen Dries and Victor